nion/swift/ExportDialog.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 # standard libraries
+from dataclasses import dataclass
```

Comment:
Can you follow the import/qualified-name convention used through the software? i.e. `import dataclasses`; then use `@dataclasses.dataclass` (i.e. fully qualified name).

In this case, `dataclass` seems unused, so can be removed entirely.

Reply:
Ok, gotcha if that is the convention you go with. Noted for future. As for it being unused, yeah I was testing out dataclasses to stretch my knowledge a bit but it ended up being a bit too isolated that way if we wanted to extend in future.  From what I could glean there didn't seem to be any major benefits for the dataclass other than some extra automatic boilerplate.